src/content/platform-integration/ios/platform-views.md

```diff
@@ -378,6 +379,7 @@ Widget build(BuildContext context) {
 ```
 
 ## Performance
+
 Platform views in Flutter come with performance trade-offs.
```

Comment:
This section on compositing is also out of date - we no longer merge the threads on iOS

Reply:
Thanks for the heads up.

I opened https://github.com/flutter/website/issues/11161 to track updating or removing the section. Might need some assistance though, as I'm not sure what (if anything) the section should discuss.

Reply:
You can delete the paragraphs from "For Example," to  "like handling OS or plugin messages."